proposal/BoWeii/README.rst

```diff
@@ -0,0 +1,115 @@
+Job shop scheduling
+===================
+
+Basic information (including the GitHub repository)
+===================
+
+Job-shop scheduling or the job-shop problem (JSP) is an optimization
+problem in computer science and operations research.
+
+I would try to use genetic-algorithm solve the JSP.
+
+**GitHub repository**: https://github.com/BoWeii/job-shop-scheduling 
+
+Problem to solve
+===================
+In a general job scheduling problem, we are given n jobs **J1, J2, ...,
+Jn** of varying processing times,each job consists of a set of
+operations **O1, O2, ..., On** which need to be processed in a specific
```

Comment:
How do you plan to let users define the operations?

Reply:
I'm sorry I don't understand what you mean, why does this mean they need to write code like me

Reply:
The key point is that for users to fill in a table, they will need to know the meaning of the values.  That requires them to understand your algorithm.  Writing code themselves is the simplest way to learn the algorithm.  That will lead them write the code themselves.

Scientists do write code, and some of them write good code.  A programmer should help them so that they do not need to learn how to code.

Reply:
Maybe I can not get your key point, users must at least convey the information they want to solve in some ways. I think filling in the form is a simple way for them to express the content of their work for a large amount. 
If not necessary, they don't need to know my algorithm, they just need a output what thay can understand.